packages/menu/src/Menu/Menu.jsx

```diff
@@ -69,14 +69,14 @@ function Menu({
         trapFocus={trapFocus}
         width={width}
         zIndex={zIndex}
-        querySelectorPortal={`.${querySelectorPortal}`}
+        querySelectorPortal={querySelectorPortal && `.${querySelectorPortal}`}
```

Comment:
this would coerce to `false`. i think we want undefined (or construct the props and omit this if we don't want it). in that regard, it's not unlike Authorization header conversations.

```
querySelectorPortal ? `.${querySelectorPortal}` : undefined
```